tower-http/CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ None.
 
 ## Breaking changes
 
-None.
+- Add `Cors` for setting [CORS] headers.
```

Comment:
i don't think adding the `Cors` middleware is a _breaking_ change?

Reply:
🤦